README.md

```diff
@@ -42,6 +42,7 @@ issue.
 1. Install a recent D compiler. DCD is tested with 2.066 and LDC 0.14.0.
 1. Run ```git submodule update --init``` after cloning this repository to grab the MessagePack and Datapacked libraries and the parser from DScanner.
```

Comment:
cool thank you. Now that i learned a bit about dub, i suggest to use dub dependencies because it may leads to better version management and more reuse/contributions to the dependencies.  On the other hand if there are any reason to keep the git submodules for now, i will try to do it with git submodules.